### ISSUE TITLE
Sort search results by players with most recent matches after relevancy

### ISSUE THREAD

**Issue by mobyte0** [odota/web#1878]
Search is useful but is harder when trying to find someone with a similar name to potentially hundreds of other players. If these results were sorted by players with the most recent match after their relevancy to the search query I think it could be beneficial.

**Comment by howardchung** [comment]
@albertcui does elasticsearch support this?

**Comment by spitfire77** [comment]
@howardchung 

we can sorting players by most recent matches with javascript 🙃

**Comment by howardchung** [comment]
yeah, but that restricts us to the max results returned from the server (150 i think)
